visallo/visallo
# fix visibility property popover not displaying for read only privilege users (4.0.1)

Testing Instructions:
- Make sure all users are able to see the visibility "i" popover

Points of Regression: property info popover

CHANGELOG
Fixed: Read only privilege users not being able to see entity level metadata
